src/ornament.js

```diff
@@ -226,7 +226,7 @@ export class Ornament extends Modifier {
     const stemDir = this.note.getStemDirection();
     const stave = this.note.getStave();
 
-    const classString = Object.keys(this.getAttribute('classes')).join(' ');
+    const classString = Object.keys(this.classes).join(' ');
```

Comment:
I don't understand this change

Reply:
> This is one refactoring that I propose in this PR: `this.attrs`
> 
> Reasons for the change:
> 
> * `this.attrs` contains three different types of information: string (`id`, `type`, ...), SVGSVGElement (`el`), Record<string, boolean> (`classes`)
> * `el`  is not used in the software, it is only initialised to null
> * there are two methods to access attributes `getAttribute` and `setAttribute`
> * there are three methods to access classes `hasClass`, `addClass`, `removeClass`
> 
> My proposal in this PR:
> 
> * Remove `el`
> * Leave string information in `this.attr` with its associated acces functions `getAttribute` and `setAttribute`.
> * Place the classes in `this.classes` with its associated access functions `hasClass`, `addClass`, `removeClass`.
> * The objects that require the complete list of classes are `annotation`, `chordsymbol` and `ornament` and they can use the protected variable `this.classes` directly.

Reply:
Thanks for the explanation Rodrigo. I agree with @wassertim that there's too much logic refactoring going on here. I think we should separate it out and focus on TypeScript.

Reply:
I suggest to close this conversation to leave have all the discussion in the first instance of `this.classes` in `annotation`